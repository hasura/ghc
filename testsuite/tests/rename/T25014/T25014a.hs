-- Should not compile as it is unclear what gets exported
module T25014a (T(MkT)) where
  import Ambig1 (T(MkT))
  import Ambig2 (T(MkT))
  data S