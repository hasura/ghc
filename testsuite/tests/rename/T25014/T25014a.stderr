T25014a.hs:2:17: [GHC-87543]
     Ambiguous occurrence ‘MkT’.
      It could refer to
         either ‘Ambig1.MkT’,
                imported from ‘Ambig1’ at T25014a.hs:3:18-23
                (and originally defined at Ambig1.hs:5:26-28),
             or ‘Ambig2.MkT’,
                imported from ‘Ambig2’ at T25014a.hs:4:18-23
                (and originally defined at Ambig2.hs:5:25-27).
     In the export: T(MkT)